src/main/java/it/gov/pagopa/pu/pagopapayments/util/ConversionUtils.java

```diff
@@ -58,4 +58,9 @@ public static OffsetDateTime localDate2RomeMaxTime(LocalDate dueDate){
       .orElse(MAX_EXPIRATION_DATE);
   }
 
+  public static LocalDateTime atEndOfDay(LocalDate localDate) {
+    return Optional.ofNullable(localDate)
```

Comment:
let's explicitly use if instead :)
let's avoid to create an Optional when possible